mpv.py

```diff
@@ -512,6 +518,29 @@ def playlist_next(self, mode='weak'):
     def playlist_prev(self, mode='weak'):
         self.command('playlist_prev', mode)
 
+    @property
+    def playlist_position(self):
+        val = self._get_property('playlist-pos')
+        return int(val) if val else None
+
+    @property
+    def playlist_length(self):
```

Comment:
Same here, ```player.playlist_count```. A more pythonic way that already works is ```len(player.playllist)``` as mpv's ```playlist``` node property is mapped to a python list of python dicts here.